yalexs/activity.py

```diff
@@ -308,7 +315,16 @@ class DoorbellBaseActionActivity(Activity):
     @cached_property
     def image_url(self):
         """Return the image URL of the activity."""
-        return self._info.get("image") or self._info.get("attachment")
+        return (
```

Comment:
It looks like this line isn't covered

Reply:
yeah a bit annoying - then it couldn't have been covered before either, right?
but i actually don't think i need the changes in this class so i'll revert these two and that should fix it